### PR TITLE
Port the cameras syncrhonization to MVVM

### PR DESCRIPTION
**Pull request by ylatuya**

### CODE REVIEW

File: LongoMatch.Drawing/Widgets/CamerasTimeline.cs [hunk 1]
```diff
@@ -33,11 +32,6 @@ namespace LongoMatch.Drawing.Widgets
 {
 	public class CamerasTimeline : SelectionCanvas, ICanvasView<ProjectVM>
```
Comment (vmartos) [inline LongoMatch.Drawing/Widgets/CamerasTimeline.cs]: Rename to CamerasTimelineView?

File: LongoMatch.Drawing/Widgets/CamerasTimeline.cs [hunk 2]
```diff
@@ -161,31 +158,31 @@ void FillCanvas ()
...
-			HeightRequest = StyleConf.TimelineCameraHeight * ViewModel.FileSet.Count ();
 		}
 
 		protected override void StartMove (Selection sel)
 		{
 			if (sel == null || sel.Drawable as TimeNodeView == null)
 				return;
 
-			(sel.Drawable as TimeNodeView).ClippingMode = NodeClippingMode.LeftStrict;
+			(sel.Drawable as TimeNodeView).ClippingMode = NodeClippingMode.NoStrict;
```
Comment (vmartos) [inline LongoMatch.Drawing/Widgets/CamerasTimeline.cs]: why this change in NodeClippingMode?

File: LongoMatch.GUI/Gui/Component/SynchronizationWidget.cs [hunk 3]
```diff
@@ -622,31 +172,30 @@ void HandleScrollEvent (object sender, System.EventArgs args)
...
-		void HandleTogglePlayEvent (TogglePlayEvent e)
+		void HandlePropertyChanged (object sender, PropertyChangedEventArgs e)
 		{
-			if (e.Playing) {
-				mainCamPlayerVM.Play ();
-			} else {
-				Pause ();
+			if (e.PropertyName == nameof (videoPlayerVM.CurrentTime)) {
+				timerule.CurrentTime = videoPlayerVM.CurrentTime;
+				camerasTimeline.CurrentTime = videoPlayerVM.CurrentTime;
```
Comment (vmartos) [inline LongoMatch.GUI/Gui/Component/SynchronizationWidget.cs]: if this only sets CurrentTime to other elements, since timerule has a videoPlayerVM as viewModel, and for camerasTimeline we can change it to accept a SyncCamVM instead of ProjectVM to get videoPlayerVM and ProjectVM. And then in the other elements subscribe to videoPlayerVM changes.

File: LongoMatch.Drawing/Widgets/CamerasTimeline.cs (hunk 2 above)
Comment (ylatuya) [inline LongoMatch.Drawing/Widgets/CamerasTimeline.cs]: The previous mode was not allowing to create positive offsets, so it was impossible to sync cameras when the secondary started before the first one.

Comment (vmartos) [inline LongoMatch.Drawing/Widgets/CamerasTimeline.cs]: 👍
